src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -272,7 +289,11 @@ def validate(self, node, options=None):
                         node.append_preceding_comment(f"PSyclone: {messages}")
                 raise TransformationError(messages)
 
-    def apply(self, node, options=None):
+    def apply(self, node, options=None, verbose: bool = False,
```

Comment:
Presumably we need to document each of these arguments now or are you planning to leave that until we remove the `options` argument?

Reply:
I'm not sure, what would your preference be? Having both `options["verbose"]` and `verbose` would be a bit too much - I'm happy to remove the `options["verbose"]` if you think just documenting the new options would be better (and would fit with deprecation maybe?).